charts/gatekeeper-artifacts/templates/crds.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.installCRDs }}
+{{- with .Values.constraintTemplateFiles }}
+{{- range $file := . }}
+    {{- $file | toYaml }}
+---
+{{- end }}
+{{- end }}
+{{- end}}
```

Comment:
This is not the way to deploy crds. Please put them in the `/crds` folder as helm3 docs say.

Reply:
Please read up on the importance of CRD deployment and how and why helm treats those as very special: https://github.com/helm/helm/issues/5871

Reply:
maybe there is a reason I am using this construct.

Reply:
in fact, we cannot even use the `/crds` folder for this chart because helm3 does not compile the templated blocks, it just renders the contents of the files.

Reply:
Ok, I can see multiple reasons why you want to do it, but that is not an explanation. If we can prevent deploying CRDs as regular resources, that is an improvement (for many reasons which I gave you before this).

CRDs are static in this context (they were probably generated in a pre-hook and thus are plain yaml when they enter this phase), so you CAN put them in the `/crds` folder.

Reply:
sorry, lot's of talk for nothing. we cannot use `/crds` folder here.

Reply:
Stop making statements like that please, it is realllllly annoying. All CRDs that we deliver are static at the moment of helm invocation, so we can just use the helm3 preferred approach.

Reply:
When this PR is ready we have to do a tools upgrade to make helmfile not need sync to deploy crds however: https://github.com/roboll/helmfile/pull/1618